acceptance/testdata/extension/commandset.txtar

```diff
@@ -0,0 +1,66 @@
+# Setup environment variables used for testscript
```

Comment:
Since this is relying on bash being available, maybe we would want to:

```
cond [exec:bash]
```

Thinking about Windows users that aren't on WSL. I'm happy to defer and have a whole other piece of work that is specific to making these tests cross-platform though, if we ever cross that bridge.

Reply:
Oof, good call. 🤔 There are Windows environments where Bash is available such as Git Bash, which should work.  That said, your recommendation is sound and will add that! ✨

Reply:
Going to leave some thoughts for the future on this:

- For gh ext testscript, since the only part where bash is actually required is for the part of running the extension.
- That portion could be refactored but obviously takes time and we're still clearing through things.
- Outside of refactoring that portion of the test out with the conditional, the only way is really switching over to creating and building a Go-based extension which is a bit more work to refactor.